modules/tinymce/src/core/main/ts/api/Resource.ts

```diff
@@ -65,6 +66,11 @@ const create = (): Resource => {
     resources[id] = data;
   };
 
+  const remove = (id: string) => {
```

Comment:
Unload is supposed to remove the ID if that is not the case then the unload function has. a bug since we introduced the resources collection. So add resources to the `unload` function instead.

- [ ] Unload should be the remove feature